bin/get-firefox

```diff
@@ -1,15 +1,16 @@
 #!/usr/bin/env node
 /* eslint-disable require-atomic-updates */
 "use strict";
-
-const index = require('..'),
-    meow = require("meow"),
-    streamToPromise = require("stream-to-promise"),
-    chalk = require("chalk"),
-    Listr = require("listr"),
-    isCI = require("is-ci"),
-    cli = meow({
-        help: `
+const _meow = import('meow');
+_meow.then((module)=> {
+     const meow = module.default;
```

Comment:
pretty sure eslint would ask to prefer destructuring if it ran.

Reply:
it did